pegasus/helpers/section_api_helpers.rb

```diff
@@ -53,13 +53,9 @@ def self.fetch_if_allowed(id_or_ids, dashboard_user_id)
 
     row = DASHBOARD_DB[:users].
       left_outer_join(:secret_pictures, id: :secret_picture_id).
-      left_outer_join(Sequel.as(:secret_words, :secret_words_1), id: :users__secret_word_1_id).
-      left_outer_join(Sequel.as(:secret_words, :secret_words_2), id: :users__secret_word_2_id).
       select(*fields,
              :secret_pictures__name___secret_picture_name,
              :secret_pictures__path___secret_picture_path,
-             :secret_words_1__word___secret_word_1,
-             :secret_words_2__word___secret_word_2
```

Comment:
no because I added it to fields (see line 58 right above and definition of fields below)